framework/framework.md

```diff
@@ -24,6 +24,39 @@ The following terms are used in this document with a specific meaning. Definitio
+* **fault tolerant time interval**: The minimum time-span from the occurrence of a *fault* to the possible occurrence of a hazardous event, if the safety mechanisms are not activated. (TODO -- Is there a better or more generic terminology for this?)
+* **inspection**: The examination of work products, following a formal procedure, to detect safety anomalies.
+* **latent fault**: A fault whose presence is not detected by a safety mechanism nor perceived by the driver within an appropriate time.
+* **malfunctioning behavior**: A *failure* or unintended behavior of an *element* with respect to its design intent.
+* **modified condition/decision coverage**: (MC/DC) A code coverage metric that indicates the percentage of single condition outcomes that independently affect a decision's outcome.  In addition, it expects that all code blocks and execution paths have been tested.
+* **quality management**: (QM) The coordinated activities to direct and control an organization with regards to quality.
+* **random hardware failure**: A failure that can occur unpredictably during the lifetime of a hardware element and that follows a probability distribution.
+* **regression strategy**: The strategy to verify that an implemented change did not affect the unchanged, existing and previously verified parts or properties of an *element*.
+* **robust design**: A design that can function correctly in the presence of invalid inputs or stressful environmental conditions.
+* **safety**: The absence of unreasonable risk.
+* **safety anomaly**: Any conditions that deviate from expectations and that can lead to harm.
+* **safety measure**: An activity or technical solution to avoid or control *systematic failures* and to detect or control *random hardware failures*, or mitigate their harmful effects
+* **software unit**: An atomic-level software *component* of the software architecture that can be subjected to stand-alone testing.
+* **system**: A set of *components* or subsystems that relates at least a sensor, a controller, and an actuator with one another. (TODO -- Normalize with the above definition)
+* **systematic failure**: A *failure* related in a deterministic way to a certain cause, that can only be eliminated by a change of the design or of the manufacturing process, operational procedures, documentation or other relevant factors.
+* **systematic fault**: A *fault* whose *failure* is manifested in a deterministic way that can only be prevented by applying process or design measures.
+* **testing**: The process of planning, preparing, and operating or exercising an *element* to verify that it satisfies specified requirements, to detect *safety anomalies*, to validate that requirements are suitable in the given context and to create confidence in its behavior.
+* **verification**: The determination whether or not an examined object meets its specified requirements.
```

Comment:
We may need a definition of *validation* as well (which is sadly lacking in ISO 26262)...